js/components/pages/Speaker.jsx

```diff
@@ -16,27 +20,36 @@ module.exports = React.createClass( {
 	},
 
 	render: function () {
-		var user = findWhere( this.props.posts.speakers, {id: Number( this.props.routeParams.id )} )
+		var user = this.props.posts.preview || findWhere( this.props.posts.speakers, {id: Number( this.props.routeParams.id )} )
 
 		if ( ! user ) {
 			return (
 				<div className="loading-wrap">
-					<div className="loading"><span className="fa fa-heart"></span> LOADING</div>
+					<div className="loading"><span className="fa fa-heart"></span> Loading</div>
```

Comment:
Just a thought: After seeing a few of the `.loading` wraps, this could be a separate react module instead. Not a blocker.